modules/dists/SparseBlockDist.chpl

```diff
@@ -433,6 +427,17 @@ class SparseBlockArr: BaseArr {
 
   proc dsiGetBaseDom() return dom;
 
+  proc sparseBulkShiftArray(shiftMap, oldnnz){
+    halt("Sparse bulk shift must be handled by local arrays");
```

Comment:
Do you mean it must be implemented in some class in particular? Or that this method should not be called?